Library/Homebrew/bintray.rb

```diff
@@ -217,8 +217,9 @@ def upload_bottles(bottles_hash, publish_package: false, warn_on_error: false)
       bintray_repo = bottle_hash["bintray"]["repository"]
       bottle_count = bottle_hash["bottle"]["tags"].length
 
-      bottle_hash["bottle"]["tags"].each do |_tag, tag_hash|
-        filename = tag_hash["filename"] # URL encoded in Bottle::Filename#bintray
+      bottle_hash["bottle"]["tags"].each do |tag, tag_hash|
+        filename = Bottle::Filename.create(bottle_hash["formula"]["name"], tag,
+                                           bottle_hash["bottle"]["rebuild"]).bintray
```

Comment:
`Filename.create` expects a formula object as the first argument. Perhaps try:

```suggestion
        filename = Bottle::Filename.new(bottle_hash["formula"]["name"], bottle_hash["formula"]["pkg_version"],
                                        tag, bottle_hash["bottle"]["rebuild"]).bintray
```